src/drafts/MarkdownEditor/_FormattingTools.tsx

```diff
@@ -1,4 +1,5 @@
-import React, {forwardRef, useImperativeHandle, useRef, useEffect} from 'react'
+import React, {forwardRef, useImperativeHandle, useRef} from 'react'
+import '@github/markdown-toolbar-element'
```

Comment:
Just wanted to double-check, I think one intent behind the condition is to allow for SSR. With this change, I believe SSR would no longer work because `@github/markdown-toolbar-element` references globals like `HTMLElement` and tries to register itself using `if (!window.customElements.get('...'))` when imported.

Reply:
That's correct. I think as we look more towards SSR we can look to centralise these dependencies into a client-side only bundle.